include/constants.h

```diff
@@ -5,9 +5,11 @@
 
 typedef u16 AnimalID;
 #define AnimalID_MUK        89
+#define AnimalID_SHELLDER   90
 #define AnimalID_STARYU     120
 #define AnimalID_STARMIE    121
 #define AnimalID_MAGIKARP   129
 #define AnimalID_MEW        151
+#define AnimalID_MAX        151
```

Comment:
I don't think this is the max animal ID actually. I think there are others in the 1000s that are sometimes used. but I get the point of making this define - maybe we could name it something else ..but no good name is coming to me

Reply:
`POKEDEX_MAX`?

Reply:
I like it. Done!